### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Not really sure with what I am dealing here 😅. I was working on Bookstack that is MinIO compatible...

### CODE REVIEW

**Comment by ericgaspar** [comment]
!tastemetitus

File: scripts/install [hunk 1]
```diff
@@ -25,385 +25,180 @@ ynh_abort_if_errors
...
-### A common way to do it is to set a weight equal to the execution time in second +1.
-### The execution time is given for the duration since the previous call. So the weight should be applied to this previous call.
-ynh_script_progression --message="Validating installation parameters..." --time --weight=1
-
-### If the app uses NGINX as web server (written in HTML/PHP in most cases), the final path should be "/var/www/$app".
-### If the app provides an internal web server (or uses another application server such as uWSGI), the final path should be "/opt/yunohost/$app"
-final_path=/var/www/$app
+ynh_script_progression --message="Validating installation parameters..." --weight=1
+
+final_path="/opt/yunohost/$app"
```
Comment (alexAubin) [inline scripts/install]: If this is a webapp I would really tend to stick to `/var/www/$app` though last time we discussed this there wasn't a 100% consensus about this é_è

**Review by alexAubin — APPROVED** [review]
Super quick review, LGTM

**Comment by Limezy** [comment]
Wow @ericgaspar how did you do to solve my console issue ?
I spent so much time on it... 
This will make my Outline installation much easier as the unit tests won't fail.
Thank you !

**Comment by ericgaspar** [comment]
Well, it passes tests, but it is not tested. I have no idear how it works

**Comment by Limezy** [comment]
Ok I'll test your version and incorporate it to my outline install script if it works. 
This should finally help me pass the tests with Outline (that I use "in prod" since 3 months now). 
Do you need some help to understand how to use or test minio ?

**Comment by tituspijean** [comment]
(Sorry, I completely missed the review request, hopefully it works alright. Thanks Eric!)

**Comment by Limezy** [comment]
There is a problem with the source : you have used a url that is regularly updated. Hence, after a few days, like now, the source is considered as corrupted because the SHA sum is not matching. I'll propose an update.

**Pull request merged**